tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/GivenUsingSuite.scala

```diff
@@ -485,31 +446,6 @@ class GivenUsingSuite extends BaseDottySuite {
     )
   }
 
-  test("using-mix-named-anonymous-error".ignore) {
```

Comment:
I removed the ignored test cases as it doesn't seem it will be easy to show sensible errors and we can focus on the happy path. Should be easy to return the test if they are needed